Assets/Scripts/EnemyPatrol.cs

```diff
@@ -0,0 +1,239 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class EnemyPatrol : MonoBehaviour
+{
+
+    [SerializeField]
+    private float moveSpeed, checkRadius, attackDelay, deathDelay, knockbackPower, knockbackHeight, knockbackTime;
+
+    public short enemyHealth;
```

Comment:
I think it's weird to have an `enemyHealth` variable that's not of the type `EnemyHealth`. Also, no public fields!